build.gradle

```diff
@@ -23,6 +23,8 @@ repositories {
 }
 
 dependencies {
+    compileOnly "org.projectlombok:lombok:1.18.4"
+    testCompileOnly "org.projectlombok:lombok:1.18.4"
```

Comment:
Is it possible to use only 'provided' here?

Reply:
> Also, provide please some example of using Lombok in code, for example create class Test and try to use lombok/delombok features

Do we need to have useless extra classes in our project? I checked it out, it worked properly. I ll show lombok's features for my team today at the presentation.

Reply:
> 
> 
> Is it possible to use only 'provided' here?

Sure, it is possible, but lombok use annotations only at compile time, we dont wanna have it in our final product.

Reply:
okay, i suppose your presentation will be enough.
Also, agree about compile time